settings.gradle

```diff
@@ -61,6 +61,7 @@ include 'clients',
     'examples',
     'generator',
     'group-coordinator',
+    'group-coordinator:group-coordinator-api',
```

Comment:
We named other internal API modules just `api`, for example `connect:api`, `storage:api`. Should we stick to the same convention?

Reply:
I just noticed we have `tools:tools-api` too

Reply:
Reusing `api` does not work. Gradle gets confused about it and dependencies fail. There is a bug in gradle. This is why we actually use `tools:tools-api` and why we have `project(":storage:api").name = "storage-api"` later in this file. So we effectively, use `module:module-api` as names expect for connect.

Reply:
For reference: https://github.com/gradle/gradle/issues/847

Reply:
@mimaison you raise a good discussion. I had the same question in creating tools-api module 😄 

As new module, maybe storage:api can be rename to storage:storage-api for consistency.